code/game/gamemodes/miniantags/abduction/abduction.dm

```diff
@@ -466,19 +466,13 @@
 	explanation_text = "You are pregnant and soon due. Find a safe place to deliver your baby."
 
 /datum/objective/abductee/engine
-	explanation_text = "Go have a good conversation with the Singularity/Tesla/Supermatter crystal. Bonus points if it responds."
-
-/datum/objective/abductee/teamredisbetterthangreen
-	explanation_text = "Tell the AI or other silicons some corny technology jokes until it cries for help."
-
-/datum/objective/abductee/licky
-	explanation_text = "You must lick anything that you find interesting."
+	explanation_text = "Go have a good conversation with the singularity/tesla/supermatter crystal. Bonus points if it responds."
 
 /datum/objective/abductee/music
-	explanation_text = "Start playing music, you're the best musician ever. If anyone hates it, beat them on the head with your instrument!"
+	explanation_text = "You burn with passion for music. Share your vision. If anyone hates it, beat them on the head with your instrument!"
 
 /datum/objective/abductee/clown
-	explanation_text = "The clown is not funny. You can do better! Make the crew laugh!"
+	explanation_text = "The clown is not funny. You can do better! Steal his audience and make the crew laugh!"
```

Comment:
You can't because multiple clowns just use their
Maybe rephrase it to `Clowns are not funny. You can do better! Steal their audience and make the crew laugh!`